packages/zengine-ui/.gitignore

```diff
@@ -1 +1 @@
-lib
+style.*
```

Comment:
@tehpsalmist maybe add forward slash here to make path explicit, we dont want to ignore all styles necesasrly just top level build artifacst

Reply:
it turns out, by omitting the slashes, this path actually means `./style.*`

Reply:
so we're good.